nvi-commons/src/main/java/no/sikt/nva/nvi/common/model/business/DbApprovalStatus.java

```diff
@@ -0,0 +1,61 @@
+package no.sikt.nva.nvi.common.model.business;
+
+import java.net.URI;
+import java.time.Instant;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbIgnore;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbImmutable;
+
+@DynamoDbImmutable(builder = DbApprovalStatus.Builder.class)
+public record DbApprovalStatus(URI institutionId,
+                               DbStatus status,
```

Comment:
Do we need to keep Db... in the name of all business objects?

Reply:
...Dao?